### ISSUE TITLE
Delete Keylogger keylogs

### ISSUE THREAD

**Issue by pj1234678** [quasar/QuasarRAT#520]
Hi Would really like to have a feature to delete logs after I'm done reading them. 
Thanks

**Comment by MaxXor** [comment]
You mean deleting the logs on the remote computer, right? As a small work-around you can use the file manager and navigate to `C:\Users\<name>\AppData\Roaming` and delete the directory called `Logs`.

**Comment by pj1234678** [comment]
Yes. I was doing that previously however it would be pretty convenient if we had a button under "Refresh logs" so we don't have to jump into the file manager every time the logs get crazy.
